templates/admin/failedJobs.tpl

```diff
@@ -0,0 +1,48 @@
+{**
+ * templates/admin/failedJobs.tpl
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Failed Jobs index
+ *}
+{extends file="layouts/backend.tpl"}
+
+{block name="page"}
+	<h1 class="app__pageHeading">
+		{translate key=$pageTitle}
+	</h1>
+	<div class="app__contentPanel">
+		<pkp-table
+			:columns="columns"
+			:rows="rows"
+			:description="description"
+			:label="label"
+            :api-url="apiUrl"
+		>
+            <template slot="actions" slot-scope="slotProps">
+                <td>
+                    <div class="buttonRow">
+                        <pkp-button @click="this.pkp.eventBus.$emit('redispatch-job', slotProps.row)">
```

Comment:
Is there a reason you used the global `eventBus` instead of the click handlers for the page? I think this could be:

```
<pkp-button @click="redispatch(slotProps.row)">
```

Reply:
Right and I dont remember why I used global `eventbus` . Probably trying out something and forgot to update it .